### ISSUE TITLE
Can't build : Parse is not found

### ISSUE THREAD

**Issue by ShawnBaek** [parse-community/ParseLiveQuery-iOS-OSX#171]
I tried..

```carthage update```

open ParseLiveQuery.workspace and build..

I got an error message, Parse is not found.

Any suggestion?

It is not work at xcode 10, swift 4.2...

**Comment by flovilmart** [comment]
Can you provide the full trace?

**Comment by ShawnBaek** [comment]
I solved this issue by clean and build again. This repo is working fine.

But I have a few problem using CocoaPods and trying to use LiveQuery in my Project

pod 'Parse'
pod 'Parse/FacebookUtils'
pod 'Parse/TwitterUtils'
pod 'ParseLiveQuery'

<img width="310" alt="screen shot 2018-10-28 at 4 22 24 am" src="https://user-images.githubusercontent.com/12643700/47608482-6c685c00-da69-11e8-9d31-0b0bc697c5e1.png">

Anyone knows how to solve this problem?

**Comment by ShawnBaek** [comment]
I found the solution

Open the Bolt-Swift -> Task.swift -> emptyTask()
Change  .success() ->  .success(())

```swift
 class func emptyTask() -> Task<Void> {
        return Task<Void>(state: .success(()))
    }
```

**Comment by TomWFox** [comment]
I believe the same problem was subsequently reported in #174 and fixed in #175, so I will close this.
